doc/src/quickstart.md

```diff
@@ -190,13 +190,18 @@ use std::io::{self, BufRead, Write};
 use lrlex::lrlex_mod;
 use lrpar::lrpar_mod;
 
-// Using `lrlex_mod!` brings the lexer for `calc.l` into scope.
+// Using `lrlex_mod!` brings the lexer for `calc.l` into scope. By default the
+// module name will be `calc_l` (i.e. the file name, minus any extensions,
+// with a suffix of `_l`).
 lrlex_mod!("calc.l");
-// Using `lrpar_mod!` brings the parser for `calc.y` into scope.
+// Using `lrpar_mod!` brings the parser for `calc.y` into scope. By default the
+// module name will be `calc_y` (i.e. the file name, minus any extensions,
+// with a suffix of `_y`).
 lrpar_mod!("calc.y");
 
 fn main() {
-    // We need to get a `LexerDef` for the `calc` language in order that we can lex input.
+    // We need to get a `LexerDef` for the `calc` language in order that we can
```

Comment:
Nitpick: Shouldn't it be `in order to lex` or just `so that we can lex`. The current sentence sounds a bit off to me.

Reply:
You're right, that is wonky.

Reply:
Try https://github.com/softdevteam/grmtools/pull/144/commits/b7c9196023f1ef9f5da4a5fa393a9ca31ed65b17.

Reply:
That works.